src/extras/geometries/CylinderBufferGeometry.js

```diff
@@ -227,23 +235,14 @@ THREE.CylinderBufferGeometry = function( radiusTop, radiusBottom, height, radial
 		for ( x = 1; x <= radialSegments; x ++ ) {
 
 			// vertex
-			vertices.setXYZ( index, 0, halfHeight * sign, 0 );
+			vertices.setXYZ( index, 0.5, halfHeight * sign, 0.5 );
```

Comment:
great question.  i think i got mixed up w/ the uv values.  corrected this and verified that the result looks good